sec/iterated.tex

```diff
@@ -49,31 +62,37 @@ \section{Iterated monotone path polytopes}
-	Indeed, in this case $P$ is the projection of a cross-polytope $T \xra{\pi} P$.
-	\cite[Theorem 5.1]{BilleraSturmfels92} says that the fiber polytope $\Sigma(T,P)$ of this projection is itself centrally symmetric.
-	A similar argument to \cref{e:zonotopes} yields the result.
-	\anibal{Is this result used in the Steenrod diagonal section? If so, consider a definition of centrally symmetric and a lemma proving this statement.}
-\end{example}
+\subsection{Zonotopes} \label{ss:zonotopes}
 
-\begin{definition}
-	We say that $(P,v)$ possess the \emph{bot-top property} if it has a unique vertex $\bm_v(P)$ which minimizes the scalar product $\angles{-,v}$ and a unique vertex $\tp_v(P)$ which maximizes it.
-	We say that $(P,\{v_k\})$ possess the \emph{bot-top property} if $(P_{k-1}, v_k)$ possess it, for every $1\leq k \leq n$.
-\end{definition}
+IF $P$ is a zonotope, then all the $P_k$ are also zonotopes.
+Indeed, in this case $P$ is the projection of a cube $C \xra{\pi} P$.
+\cite[Theorem 4.1]{BilleraSturmfels92} says that the fiber polytope $\Sigma(C,P)$ of this projection is itself a zonotope, the \emph{fiber zonotope} of $P$.
+We consider the flag $C \xra{\pi} P \xra{\pi_1} I$, where $I$ is the $1$-dimensional image of $P$ under the projection $\pi_1$.
+Using the functoriality property \cite[Lemma 2.3]{BilleraSturmfels92}, we have that $\Sigma(P,I)=\pi(\Sigma(C,I))$ is the projection of a zonotope, thus itself a zonotope.
+Repeating this argument, we get that all the $P_k$ are zonotopes.
+\anibal{What is a zonotope? Why are they interesting?}
+
+\subsection{Centrally symmetric polytopes} \label{ss:centrally-symmetric}
+
+If $P$ is centrally symmetric, then all the $P_k$ are also centrally symmetric.
+Indeed, in this case $P$ is the projection of a cross-polytope $T \xra{\pi} P$.
+\cite[Theorem 5.1]{BilleraSturmfels92} says that the fiber polytope $\Sigma(T,P)$ of this projection is itself centrally symmetric.
+A similar argument to \cref{ss:zonotopes} yields the result.
+\anibal{Is this result used in the Steenrod diagonal section? If so, consider a definition of centrally symmetric and a lemma proving this statement.}
+
+\subsection{Bot-top property}
+
+\anibal{I am still thinking of a different name for this}
+
+We say that $(P,v)$ possess the \emph{bot-top property} if it has a unique vertex $\bm_v(P)$ which minimizes the scalar product $\angles{-,v}$ and a unique vertex $\tp_v(P)$ which maximizes it.
+We say that $(P,\{v_k\})$ possess the \emph{bot-top property} if $(P_{k-1}, v_k)$ possess it, for every $1\leq k \leq n$.
 
 The bot-top property for $(P,v)$ was named \emph{quasi-orientation} in \cite[Definition 1.11]{GLA21}.
 
+\subsection{Standard globe}
+
+\anibal{In the Steenrod section you have a piecewise linear map from the cube, which is appealing because of the rigidity of such maps. Maybe the statement is that it factors through the/a projection from the cube to the standard globe?}
```

Comment:
Good question. Every time we have two maps from the n-cube, I imagine that they could be assembled to a map from the cube itself. Is this a result you would like to have? You mentioned a particular map between the cube and the globe, a few weeks ago